bitnami/harbor/Chart.yaml

```
@@ -55,4 +55,4 @@ maintainers:
 name: harbor
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/harbor
-version: 24.1.8
+version: 24.1.9
```

Please bump minor version instead given you're adding new parameters:
```suggestion
version: 24.2.0
```